#cluster elements
float64 size
cluster_point[] pt
